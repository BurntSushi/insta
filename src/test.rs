#[test]
fn test_embedded_test() {
    assert_snapshot_matches!("embedded", "Just a string");
}