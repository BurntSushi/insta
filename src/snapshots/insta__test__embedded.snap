---
source: src/test.rs
assertion_line: 3
expression: "\"Just a string\""
---
Just a string
